packages/sth-config/src/config-service.ts

```diff
@@ -20,6 +20,10 @@ const _defaultConfig: STHConfiguration = {
             exposePortsRange: [30000, 32767],
             hostIp: "0.0.0.0"
         },
+        runnerImages: {
+            python3: "",
+            node: "",
+        },
```

Comment:
I am wondering whether we should remove the `image` key from runner & prerunner configurations. What do you think?

Reply:
Yeah. I consulted with Ala and she thinks it's good enough for now, and should be addressed more thoroughly later.